templates/partials/navbar.njk

```diff
@@ -42,7 +42,7 @@
           </ul>
         </li>
         <li class="dropdown navbar-inverse">
-          <a href="#" class="dropdown-toggle programs-nav" data-toggle="dropdown"><i class="fa fa-tasks hidden-xs hidden-sm"></i> Programs</a>
+          <a href="#" class="dropdown-toggle programs-nav" data-toggle="dropdown"><i class="fas fa-tasks"></i> Programs</a>
```

Comment:
i think this might be where the icons are staying present on smaller/in-between sized devices which doubles up the nav/header bar